index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="Top">
+            <nav class="menu">
+                <img class="logo" src="images/toolbar-mobile.png" alt=""><a href="#"></a>
+            </nav>
+         </header>
+        <h1>I'm Jorum</h1>
+        <h2>Glad to see you!</h2>
+    
+        <div class="details">
+            <p>I am a software developer! I can help you build a product.,
+                feature or website. look through some of my work and experience!
+                If you like what you see and you have a project you need coded,
+                do not hesitate to contact me.
+            </p>
+        </div>
+        <div class="connect">
+            <p>LET'S CONNECT</p>
+        </div>
+        <div class="connect-logo">
+            <ul>
+                <li><a href="#"><img class="twitter" src="images/twitterIcon.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/linkedInIcon.png" alt=""></a></li>
+                <li><a href="#"><img src="images/mIcon.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/gitHubIcon.svg" alt=""></a></li>
+                <li><a href="#"><img src="images/peaceIcon.svg" alt=""></a></li>
+            </ul>
+        </div>
+    </div>    
```

Comment:
As asked by the previous reviewer you should use a `<section>` tag for this is the headline section for semantic reasons.

```
<div class=container>
   <header>...</header>
    <section class="hero">.put your code here :  h1, h2, and the 3 .</section>
    <section class="mywork">next section</section>
</div>
```